.gitignore

```diff
@@ -1 +1,6 @@
+### Helm ###
+# Chart dependencies
+**/charts/*.tgz
```

Comment:
This is technically wrong, as adding dependencies directly in the `charts/` folder is a perfectly valid things to do. You can even reference local dependencies in `charts/` in the `Chart.yaml`. 
I guess this is mostly for preservation purposes. 
But I have never had the actual usecase for it, so I am happy to have that in the ignore (although, it may be that then `Helm` complains about the `lock` file being out of sync :thinking: )